.github/workflows/validate.yml

```diff
@@ -84,10 +84,10 @@ jobs:
           EXIT_VALUE=0
 
           # external contributors can modify some files
-          EXTERNAL_REGEX="^components\/(pin|i2c|servo|ds18x20|pwm|pixel|uart)\/.*\/(definition\.json|image\.(png|jpe?g|gif))$"
+          EXTERNAL_REGEX="^components\/(pin|i2c|servo|ds18x20|pwm|pixel|uart|i2c_output)\/.*\/(definition\.json|image\.(png|jpe?g|gif))$"
```

Comment:
I think it might make more sense to keep it under i2c because what happens when we have a mixed input and output i2c device like the DACs or seesaw stuff (iot neopixel button, or seesaw analog potentiometer slider)

Reply:
I am not sure I agree here. The i2c output components do not completely "fit" the standard I2C component form. There is no periodic or event-driven polling, they provide _options_ for configuration instead.

Reply:
Agreed, I meant that we will have in future components that include both output and input, so do we keep the idea of that being an i2c sensor plus an i2c_output as one "user component"?